CardanoSharp.Wallet/CIPs/CIP2/CoinSelectionUtility.cs

```diff
@@ -10,18 +10,18 @@ namespace CardanoSharp.Wallet.CIPs.CIP2
 {
     public static class CoinSelectionUtility
     {
-        public static CoinSelection UseLargestFirst(this TransactionBodyBuilder tbb, List<Utxo> utxos)
+        public static CoinSelection UseLargestFirst(this TransactionBodyBuilder tbb, List<Utxo> utxos, string changeAddress, ITokenBundleBuilder mint = null, int limit = 20, ulong feeBuffer = 0)
         {
-            var cs = new CoinSelectionService(new LargestFirstStrategy(), new SingleTokenBundleStrategy());
+            var cs = new CoinSelectionService(new LargestFirstStrategy(), new BasicChangeSelectionStrategy());
             var tb = tbb.Build();
-            return cs.GetCoinSelection(tb.TransactionOutputs.ToList(), utxos);
+            return cs.GetCoinSelection(tb.TransactionOutputs.ToList(), utxos, changeAddress, mint, limit, feeBuffer);
         }
 
-        public static CoinSelection UseRandomImprove(this TransactionBodyBuilder tbb, List<Utxo> utxos)
+        public static CoinSelection UseRandomImprove(this TransactionBodyBuilder tbb, List<Utxo> utxos, string changeAddress, ITokenBundleBuilder mint = null, int limit = 20, ulong feeBuffer = 0)
         {
-            var cs = new CoinSelectionService(new RandomImproveStrategy(), new SingleTokenBundleStrategy());
+            var cs = new CoinSelectionService(new RandomImproveStrategy(), new BasicChangeSelectionStrategy());
             var tb = tbb.Build();
-            return cs.GetCoinSelection(tb.TransactionOutputs.ToList(), utxos);
+            return cs.GetCoinSelection(tb.TransactionOutputs.ToList(), utxos, changeAddress, mint, limit, feeBuffer);
         }
     }
-}
```

Comment:
the purpose of this class may not be clear. its suppose to help abstract away the configuring of different input and change strategies. so we dont need to really update unless parameters are different. really we just add to this.

Reply:
My reason for updating is the SingleTokenBundle strategy will creating failing transactions for wallets with large amounts of NFTs. I haven't had any issues with the new strategy so wanted to make sure anyone using this function doesn't have failing tx's unexpectedly